app/mailers/mailer.rb

```diff
@@ -55,4 +55,13 @@ def mfa_notification(user_id)
     mail to: @user.email,
       subject: "Please consider enabling MFA for your account"
   end
+
+  def gem_yanked(yanked_by_user_id, version_id, notified_user_id)
+    @version        = Version.find(version_id)
+    notified_user   = User.find(notified_user_id)
+    @yanked_by_user = User.find(yanked_by_user_id)
+
+    mail to: notified_user.email,
+         subject: "Gem #{@version.to_title} yanked from RubyGems.org"
```

Comment:
`I18n.t`? I see it as used almost everytime (except mfa_notification, which we should change to use it as well IMHO).

Reply:
as of now, we don't support i18n for mails. we need to start storing user preference in db, until then adding keys seemed an unnecessary hassle to me.
I can change if you think we should use i18n.t anyways.

Reply:
`I18n` is handy also to just keep code out of user facing texts. Anyway not blocker IMHO.